### PR TITLE
Describe how to add a tracking number to a shipment.

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes how to add a tracking number to a shipment.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/sales/shipments-create.html

### CODE REVIEW

File: src/sales/shipments-create.md [hunk 1]
```diff
@@ -28,14 +28,45 @@ For each line item in the order, modify the **Qty to Ship** as needed.
...
+
+1. Scroll down to the _Payment & Shipping Method_ block.
 
 1. Select the **Carrier**.
 
 1. Enter a **Title** for the package.
 
 1. Enter the tracking **Number**.
 
+    Tracking the shipment is an important point in management. To track the location of any shipment open the required shipment in edit mode and click **Track this shipment**.
```
Comment (dobooth) [inline src/sales/shipments-create.md:65]:
```suggestion
    To track the location of any shipment, open the required shipment in edit mode and click **Track this shipment**.
```

File: src/sales/shipments-create.md [hunk 2]
```diff
@@ -28,14 +28,45 @@ For each line item in the order, modify the **Qty to Ship** as needed.
...
+
+1. In the **Action** column for the selected order, click **View**.
+
+1. Click **Ship**.
+
+1. Scroll down to the _Payment & Shipping Method_ block and click **Add Tracking Number**.
+
+1. Set **Carrier**:
+
+    - Custom Value;
```
Comment (dobooth) [inline src/sales/shipments-create.md:43]: I think we can remove the punctuation at the end of these list items.

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/sales/shipments-create.md (hunk 2 above)
Comment (ZhovnirMaria) [reply src/sales/shipments-create.md]: thanks, done ✅

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**